eqc/event_logger.erl

```diff
@@ -128,6 +128,6 @@ add_event(#event{timestamp = Now, data = Data}, State) ->
   State#state{ events = [Event|State#state.events] }.
 
 timestamp() ->
-  {A, B, C} = erlang:now(),
+  {A, B, C} = os:timestamp(),
```

Comment:
Alternatively `erlang:system_time(microseconds).` which is slightly shorter

Reply:
Switched to `erlang:system_time(microsecond)`